traitsui/qt4/toolkit.py

```diff
@@ -451,238 +451,3 @@ def font_trait(self, *args, **traits):
         from . import font_trait as ft
```

Comment:
I imagine all these factory methods for trait types can go as well?  I'm guessing that will be a follow up PR?

Reply:
ah maybe not https://github.com/enthought/traitsui/blob/master/traitsui/toolkit_traits.py

Reply:
exactly. the toolkit traits are used and exposed to end users. I'll evaluate the other methods on the toolkit object later.